ansible_ai_connect/ai/api/model_pipelines/nop/pipelines.py

```diff
@@ -143,3 +146,16 @@ def invoke(self, params: ChatBotParameters) -> ChatBotResponse:
 
     def self_test(self) -> Optional[HealthCheckSummary]:
         raise NotImplementedError
+
+
+@Register(api_type="nop")
+class NopStreamingChatBotPipeline(NopMetaData, ModelPipelineStreamingChatBot[NopConfiguration]):
```

Comment:
Is this class used anywhere?

Reply:
This class did not exist in early PoC code and I think Streaming chat works without this class. 
I added this only for fixing a test failure.

Reply:
I think for when the feature is not implemented/enabled and the API is being called. The tests must be testing this case - good testing framework there! So we need this.